docs/conf.py

```diff
@@ -591,3 +591,12 @@ def configure_doxyfile(
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #
 # texinfo_no_detailmenu = False
+
+# -- Copy the modules documentation
+
+from urllib.request import urlretrieve
+
+urlretrieve (
+    "https://raw.githubusercontent.com/eProsima/Fast-DDS/master/RELEASE_SUPPORT.rst",
+    "notes/release_support.rst"
+)
```

Comment:
I'm thinking that another way would be to try to use the one that Fast DDS installs (we'd need to install it on Fast of course)

Reply:
First, trying to avoid to install a `rst` file was something we discussed internally. Second, I do not think that maintaining different versions of this file is going to be of interest to the user. The user can access directly all the minor releases and its lifecycle independently of the release being used.

Reply:
After discussing with the team, we will proceed with adding simply a link in the documentation (some concern about not being able to download the rst file were presented). Also, only one version of the file is going to be maintained, the one in Fast DDS `master` branch.